src/main/java/com/team1/dealerApp/entities/Movie.java

```diff
@@ -0,0 +1,28 @@
+package com.team1.dealerApp.entities;
+
+import com.team1.dealerApp.models.Genre;
+import com.team1.dealerApp.models.Video;
+import com.team1.dealerApp.models.VideoStatus;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import java.time.Year;
+import java.util.List;
+
+@Entity
+@Setter
+@Getter
+@NoArgsConstructor
+@SuperBuilder
+public class Movie extends Video {
+
+    @Column(name="running_time", nullable = false)
+    private int runningTime;
+
+    public Movie(Long id, String title, Genre genre, List<String> cast, String director, Year releaseYear, double purchasePrice, double rentalPrice, String plot, float rating, VideoStatus videoStatus, int runningTime) {
```

Comment:
Come mai hai scritto il costruttore a mano invece di usare la annotation di Lombok?

Reply:
Visto che la classe eredita il costruttore e gli attributi da Video, Lombok con @AllArgsConstructor restituisce un costruttore con solo gli attributi della classe in cui è presente l'annotation.
Ho ovviato a questa cosa con il costruttore esplicito con tutti i campi necessari.